packages/react-components/src/components/data-providers/tsp-data-provider.ts

```diff
@@ -139,24 +139,22 @@ export class TspDataProvider {
         };
     }
 
-    async getArrows(ids: number[], viewRange?: TimelineChart.TimeGraphRange, nbTimes?: number): Promise<TimelineChart.TimeGraphArrow[]> {
+    async getArrows(viewRange?: TimelineChart.TimeGraphRange, nbTimes?: number): Promise<TimelineChart.TimeGraphArrow[]> {
```

Comment:
We don't use ids for arrows because we need all the arrows for the time range, some arrows might cross the visible area but start and end at not visible rows.